annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -20,9 +20,9 @@ You can find the generated Helm artifacts under: `target/classes/META-INF/dekora
 
 How can we use it?
 
-First, make sure you have logged into a cluster with Helm enabled (in OpenShift clusters, Helm is enabled by default).
+First, make sure you have installed [the Helm command line](https://helm.sh/docs/intro/install/) and have connected into a cluster.
```

Comment:
I suggest to say instead of `and have connected into a cluster.` --> `and has access to a kubernetes cluster`